src/gam.py

```diff
@@ -32,7 +32,7 @@
 import csv
 import datetime
 import hashlib
-import http.client
+import http.client as http_client
```

Comment:
thanks for catching. This is fine for now though we should probably fix the issue by not using http as a var name at some point.